app/models/note.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: notes
+#
+#  id         :bigint(8)        not null, primary key
+#  title      :string
+#  content    :string
+#  note_type  :string
+#  user_id    :bigint(8)        not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Note < ApplicationRecord
+  validates :user_id, :title, :content, :note_type,
+            presence: true
+  validates :note_type, inclusion: { in: %w[review critique] }
```

Comment:
Muy bien esta validación! En realidad esto solemos resolverlo con enums. Te dejo un post para chusmear como usarlo: https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/